pkg/handler/v2/external_cluster/aks.go

```diff
@@ -651,9 +651,21 @@ func createMachineDeploymentFromAKSNodePoll(nodePool *armcontainerservice.Manage
 	if nodePool.UpgradeSettings != nil {
 		md.Cloud.AKS.Configuration.MaxSurgeUpgradeSetting = to.String(nodePool.UpgradeSettings.MaxSurge)
 	}
-	if nodePool.ProvisioningState != nil && (nodePool.PowerState != nil || nodePool.PowerState.Code != nil) {
+	if nodePool.ProvisioningState != nil && (nodePool.PowerState != nil && nodePool.PowerState.Code != nil) {
 		md.Phase = apiv2.ExternalClusterMDPhase{
 			State: aks.ConvertMDStatus(*nodePool.ProvisioningState, *nodePool.PowerState.Code),
+			AKS: &apiv2.AKSMDPhase{
+				ProvisioningState: apiv2.AKSProvisioningState(*nodePool.ProvisioningState),
+				PowerState:        apiv2.AKSPowerState(*nodePool.PowerState.Code),
+			},
+		}
+		switch {
+		case string(*nodePool.PowerState.Code) == string(resources.StoppedAKSMDState) && *nodePool.ProvisioningState == string(resources.SucceededAKSMDState):
```

Comment:
message could be something like this:
`The cluster and node pools are in stopped.
You can only start or delete a stopped AKS cluster. To perform any operation like scale or upgrade, start your cluster first.`